docs/getting-started/how-prisma-schemas-are-converted-into-entities.md

```diff
@@ -0,0 +1,283 @@
+#### Model IDs  
+
+As stated in the [warnings section](#log-warnings), users cannot use composite IDs in their models due to current limitations with Amplication.
+
+This phase converts `@@id` attributes in your models to `@@unique` attributes.
+
+It also introduces an `id` field of type `String` to every model that uses a composite ID.
+
+Here's an example of this transformation:
+
+```graphql title="schema.prisma"
+model DomainUnit {
++id  String @id @default(cuid())
+  domainId String
+  unitId   String
+
+-@@id([domainId, unitId])
++@@unique([domainId, unitId])
+}
+```
+
+#### The ID field  
+
+Finally, there's a conversion of all ID fields.
+
+This part ensures that your Prisma schema aligns with Amplication's requirement of having an **`id`** field for each model.
+
+non-ID fields named **`id`** are renamed to **`${modelName}Id`** to prevent any collisions with the actual ID field.
+
+If an ID field (a field with an `@id` attribute) has a different name, it's renamed to **`id`**.
+
+But, in either case, a `@map` custom attribute is added to the field with the original field name.
+
+Here's an example:
+
+```diff title="schema.prisma"
+model RandomName {
+-randomId.         Int     @id  
++id                Int     @id @map("randomId")
+-id.               Int?
++randomNameId     Int?    @map("id")
+  fullName         String
+} 
+```
+
+### Step 3: Convert  
+
+Following the preparation phase, we leverage the updated Prisma schema to generate Amplication's entity and field objects derived directly from this prepared schema.
+
+The first thing that happens is that the Prisma Models get converted into Amplication entities.
+This is based on the following information:
+
+- Name
+- Display Name
+- Plural Display Name
+- Description
+- Custom Attribute
+- Fields
+
+The next thing that happens is that the Prisma data types get converted into Amplication data types.
+
+Here is a re-written table summarizing the Prisma to Amplication data type conversions:
+
+| Prisma | Amplication | Description |
+|-|-|-|  
+| String (ID) | Id | Converts ID field with String type to Amplication Id type, defaults to cuid |
+| Int (ID) | Id (autoincrement) | Converts ID field with Int type to autoincrement Amplication Id |   
+| Boolean | Boolean | Same Boolean type |
+| DateTime (@default(now())) | CreatedAt | DateTime with now() default converts to CreatedAt |
+| DateTime (@updatedAt) | UpdatedAt | DateTime with updatedAt attribute converts to UpdatedAt |
+| DateTime | DateTime | Plain DateTime remains DateTime |
+| Float, Decimal | Decimal | Float and Decimal convert to Decimal |
+| Int, BigInt | WholeNumber | Int and BigInt convert to WholeNumber |
+| String | SingleLineText | String converts to SingleLineText |    
+| Json | Json | Json types are equivalent |  
+| Enum | OptionSet | Enum converts to OptionSet |
+| Enum[] | MultiSelectOptionSet | Enum array converts to MultiSelectOptionSet |
+| Model relation | Lookup | Relation to another model becomes Lookup |
```

Comment:
@abrl91 we need to update the list with the new types

Reply:
https://github.com/amplication/docs/pull/317/files/9c503dae0ef7d23052733038a7b60b4842cb9f79#r1294641455